sys/include/net/conn/udp.h

```diff
@@ -105,9 +105,9 @@ int conn_udp_recvfrom(conn_udp_t *conn, void *data, size_t max_len, void *addr,
 /**
  * @brief   Sends a UDP message
  *
- * @param[in] data      Pointer where the received data should be stored.
- * @param[in] len       Maximum space available at @p data.
- * @param[in] src       The source address. May be NULL for all any interface address.
+ * @param[in] data      Pointer to the payload.
```

Comment:
Data that should be put into the packet. The term payload is layer agnostic so it shouldn't be a problem that someone does not understand UDP. Payload can be plain user data or a home-brewed coap header. I see no need to explain `payload` further in this context.